_assets/scss/modules/_projects.scss

```diff
@@ -25,6 +25,18 @@
     margin-bottom: 0;
     // @include col-third(false);
   }
+
+  h4 {
+    @include h4--med;
+    line-height: $unit * .875;
+    color: $black;
```

Comment:
After with andre $black should be the default and we should change it here: https://github.com/rationaledesign/rationale/blob/master/_assets/scss/elements/_base.scss#L13

Which would obviously mean colors will need to change because of this.